agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -136,6 +159,7 @@ def start_acq(self, session, params=None):
                     'block_name': 'pressures',
                     'data': {}
                 }
+                self.gauge.channel_power()
```

Comment:
Currently this will generate a "Something is off!" log message at 2.5 Hz if conditions are met. I would suggest perhaps caching channel states in the `Pfeiffer` object, and logging state changes only when they change, rather than on every iteration of the loop.

Reply:
Will the `txaio` module accomplish this?

Reply:
Not by itself, it just replaces the print statement. You'd have to still cache the data and check if the state has changed.